Tests/SmithyIdentityTests/StaticAWSCredentialIdentityResolverTests.swift

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
This test was moved from aws-sdk-swift to here, as the thing it tests was moved here.